frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -164,6 +165,28 @@ const plugin: Plugin<ConsumedExtensions> = [
       query: `${CAPACITY_USAGE_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_USED]}[60m:5m]`,
     },
   },
+  {
+    type: 'Dashboards/Storage/TopConsumers/Item',
+    properties: {
+      name: 'VMs',
+      queryNamePrefix: 'VMS_',
+      queryNameSuffix: 'BY_USED',
```

Comment:
, I'd split the extensions to two types, instead of  `type: 'Dashboards/Storage/TopConsumers/Item'`, you would have `type: 'Dashboards/Storage/TopConsumers/Requested` and `type: 'Dashboards/Storage/TopConsumers/Used`. There would be no need to specify `queryNameSuffix` then.